lmp/bb-build.sh

```diff
@@ -1,5 +1,36 @@
 #!/bin/bash -e
 
+function finish() {
+    # we need to check that because it is not available before kirkstone
+    if command -v bitbake-getvar >/dev/null 2>&1; then
+        # get buildstats path
+        BUILDSTATS_PATH="$(bitbake-getvar --value TMPDIR | tail -n 1)/buildstats"
+    fi
+    # check if the buildstats was enabled
+    if [ -d "$BUILDSTATS_PATH" ]; then
+        # get the most recent folder
+        BUILDSTATS_PATH="$(ls -td -- $BUILDSTATS_PATH/*/ | head -n 1)"
+        # we need to check that because it can't be available in old containers
+        if command -v xvfb-run >/dev/null 2>&1 ; then
+            # producing bootchart.svg
+            run xvfb-run ../layers/openembedded-core/scripts/pybootchartgui/pybootchartgui.py \
```

Comment:
Wouldn't it be better to use TOPDIR here instead of relative addressing? `$TOPDIR/../layers/...`

Reply:
`TOPDIR` only exist inside the bitbake environment, what we have on bash shell is the `BUILDDIR` that points to the build directory.
Anyway because the `BUILDDIR` was allways created inside the `OEROOT` the layers path is also allways rechable using the `../layers/`
https://github.com/foundriesio/lmp-manifest/blob/427d603fdf5703fee7922509cf947f72f72cda74/setup-environment-internal#L123-L127

Reply:
I see. I meant `bitbake-getvar --value TOPDIR`, not the env var :)

But if BUILDDIR is available directly, it would be better to use `$BUILDDIR/../layers/...`

The point is to avoid using relative paths.

Reply:
Fixed. thanks